Pulling in utmpx.h which does not have UT_HOSTSIZE, UT_LINESIZE & UT_NAMESIZE defined.

Causing:
w.c: In function 'print_from':
w.c:236:43: error: 'UT_HOSTSIZE' undeclared (first use in this function)
  236 |    print_display_or_interface(u->ut_host, UT_HOSTSIZE, fromlen - len);
      |                                           ^~~~~~~~~~~
w.c:236:43: note: each undeclared identifier is reported only once for each function it appears in
w.c: In function 'getproc':
w.c:357:12: error: 'UT_NAMESIZE' undeclared (first use in this function)
  357 |   char buf[UT_NAMESIZE + 1];
....
diff --git a/src/w.c b/src/w.c
index 5e878f0..568db02 100644
--- a/src/w.c
+++ b/src/w.c
@@ -46,11 +46,7 @@
 #include <termios.h>
 #include <time.h>
 #include <unistd.h>
-#ifdef HAVE_UTMPX_H
-#	include <utmpx.h>
-#else
-#	include <utmp.h>
-#endif
+#include <utmp.h>
 #include <arpa/inet.h>
 
 #include "c.h"
